packages/p2p/src/components/orders/order-table/order-table-content.jsx

```diff
@@ -68,7 +68,10 @@ const OrderTableContent = () => {
     }
 
     return (
-        <Empty has_tabs icon='IcNoOrder' title={localize('You have no orders.')}>
+        <Empty has_tabs icon='IcNoOrder'>
+            <Text size='s' weight='bold'>
+                <Localize i18n_default_text='You have no orders.' />
+            </Text>
             {general_store.is_active_tab && (
```

Comment:
Ditto

Reply:
Hello Matin, Mustofa suggested that I should change to the different Localize function in case the latter doesn't work. Although from my understanding, the issue seems to be resolved or not reproducable anymore, which I think make these changes I made obsolete.